python/tank/platform/qt/login.ui

```diff
@@ -0,0 +1,228 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>LoginDialog</class>
+ <widget class="QDialog" name="LoginDialog">
+  <property name="windowModality">
+   <enum>Qt::ApplicationModal</enum>
+  </property>
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>374</width>
+    <height>324</height>
+   </rect>
+  </property>
+  <property name="styleSheet">
+   <string notr="true">QWidget
```

Comment:
Maybe we should do a separate quick chat about the UI. As we are lifting this across from the login framework, I think all this special style should be taken out so that the login UI is consistent with the the rest of toolkit UX.